qiskit_experiments/database_service/db_fitval.py

```diff
@@ -34,3 +34,11 @@ def __str__(self):
         if self.unit is not None:
             out += f" {str(self.unit)}"
         return out
+
+
+def __new__(cls, *args, **kwargs):
+    warnings.warn(
+        "FitVal object has been deprecated in Qiskit Experiments version 0.3 and "
+        "will be removed in version 0.5. Use version <= 0.3 to load this object.",
+        DeprecationWarning,
+    )
```

Comment:
`__new__` has to return an object, also the indent is wrong so it's probably not actually being added to the class.

Reply:
Since this is deprecated, we should probably also add handling for loading past results containing FitVals and converting them to UFloats so that when you re-save them it won't raise deprecation warnings

Reply:
Sorry this is my mistake. I tried to keep this as serialization format with another PR, but we could remove this.